### ISSUE TITLE
[ISSUE] NumberFormatException on Custom Duration Rest Period

### ISSUE THREAD

**Issue by davcpas1234** [VersCreativeUK/FACETS#25]
**Application Version**

0.0.5

**Issue Description**

When entering a custom duration and confirming a crash is observed.

**Steps to Reproduce**

1. Go to Rest Sleep/Wake Routine
2. Click on '+' under Rests
3. Click 'Custom Duration...' Chip
4. Enter a duration
5. Click 'Ok'
6. Crash

**Stack Trace**
```
java.lang.NumberFormatException: For input string: "Custom Duration..."
        at java.lang.Integer.parseInt(Integer.java:615)
        at java.lang.Integer.parseInt(Integer.java:650)
        at uk.ac.bournemouth.facets.Fragments.RestSleepWakeRoutine.RSWRoutineAddRestFragment$5$override.onCheckedChanged(RSWRoutineAddRestFragment.java:262)
```

**Expected Behaviour**

No crash - Custom duration is formatted and replace "Custom Duration" Text

**Comment by davcpas1234** [comment]
Added additional conditional change clause to ignore custom duration choice from other option duration calculation.

```lang:java
Chip customDurationChip = view.findViewById(R.id.durationCustom);
ChipGroup durationChipGroup = view.findViewById(R.id.rest_period_duration_chips);
durationChipGroup.setOnCheckedChangeListener(new ChipGroup.OnCheckedChangeListener() {
    @Override
    public void onCheckedChanged(ChipGroup chipGroup, int i) {
        Chip chip = chipGroup.findViewById(chipGroup.getCheckedChipId());
        if(chip != null) {
            if(chip != customDurationChip) { // <-- Fix here
                durationSelected = Integer.parseInt(chip.getText().toString().split(" mins")[0]) * 60000;
            }
        }else{
            Log.e(TAG, "Error selecting chip.");
        }
    }
});
```

**Comment by davcpas1234** [comment]
Released in `0.0.6`
